packages/consent-manager/util/cookies.js

```diff
@@ -49,4 +58,7 @@ export function savePreferences(prefs, version) {
     expires: COOKIE_EXPIRES,
     domain,
   })
+  preferencesLoaded = true
 }
+
+export const preferencesSavedAndLoaded = () => !!preferencesLoaded
```

Comment:
Optional 🎨 If we default `preferencesLoaded` to `false` when it's initialized above `getDomain`, I think we'd just be able to return the variable from this function without the `!!`.

Reply:
Good point! I had it as undefined originally when writing some tests but we can go this route now 👍